R/liftover.R

```diff
@@ -0,0 +1,287 @@
+#' @title Liftover Bedpe.
+#'
+#' @description Use liftOver to convert a bedpe file between the two main genome builds (grch37/hg38).
+#'
+#' @details The user can specify a path to the bedpe file that needs to be lifted with `bedpe_file`,
```

Comment:
Update this documentation to also state what other data types can be "lifted". Possibly also how a user can obtain them (e.g `maf = get_ssm_*`, `bedpe = get_manta_sv`, `bed = bundled bed objects`)

Reply:
Updated!